telethon/crypto/aes.py

```diff
@@ -1,6 +1,6 @@
 import os
 import pyaes
-
+from . import libssl
```

Comment:
Haven't tested but this will:
```python
raise ImportError('libssl.so not found')
```

If the `'ssl'` library is not found, correct? Shouldn't this rather be:
```python
try:
    from . import libssl
except ImportError:
    libssl = None
```

And then below check `if libssl`?